app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -68,7 +68,11 @@
 
 <%= render LanguageSkillsComponent.new(application_form: @application_choice.application_form) %>
 
-<%= render PersonalStatementComponent.new(application_form: @application_choice.application_form) %>
+<% if @application_choice.application_form.continuous_applications? %>
+  <%= render ChoicePersonalStatementComponent.new(application_choice: @application_choice) %>
+<% else %>
+  <%= render PersonalStatementComponent.new(application_form: @application_choice.application_form) %>
+<% end %>
```

Comment:
👍

Reply:
Is there a reason behind creating a new component? I am okay by the way of using the new component I just want to understand the reasoning. The good thing about creating new ones is that we can delete the old ones :). Is that the reason?

Reply:
Iain and I spoke about this yesterday – the `PersonalStatementComponent` is being used on both the support and provider UI. Changing this to render just the application choice personal statement makes sense for Manage, but not such a great experience for support.

We'll turn our attention to the support UI once we've launched in the new cycle, we'll update the support application choice component to include the personal statement content